### ISSUE TITLE
Add "alert me when product is available" feature(?).

### ISSUE THREAD

**Issue by aubrel** [meitar/flora-posidonia#17]
Because multiple people have already requested the same product (in person, and likely will do so through the custom order form), it may be a good idea to add a feature on the [[Home]] page that states which products are currently "in production" (in so doing, this will save the customer the trouble of submitting a custom order, which, by the time this product is already "in production," will be unnecessary -- a product will be added to the regular catalogue if it is requested 3 times or more, either in person or through the custom order form). Also, including a "send me an alert when this product is available for purchase" option would be good. 

In other words, in this case, a product will have been requested 3 or more times, and thus will be marked for inclusion to the permanent catalogue (by adding a product to WooCommerce which is invisible at first and then made public upon completion). Because this product has _already been marked for inclusion to the catalogue,_ any _additional_ custom orders will be unnecessary; the product is already in larger scale production and will be available for anyone else who wants that product. Those people can then simply say "alert me when this is done," and will be pinged once the product is available for regular purchase.
